sunpy/map/header_helper.py

```diff
@@ -361,3 +362,54 @@ def _set_instrument_meta(meta_wcs, instrument, telescope, observatory, wavelengt
     'CD2_2':
     'Matrix element CDi_j describing the rotation required to align solar North with the top of the image.'
 }
+
+
+def carrington_header(date, observer_coordinate, shape, *, projection_code="CAR"):
+    """
+    Construct a FITS-WCS header for a Carrington coordinate frame.
+    The date-time and observer coordinate of the new coordinate frame
+    are taken from the input map. The resulting WCS covers the full surface
+    of the Sun, and has a reference coordinate at (0, 0) degrees Carrington
+    Longitude/Latitude.
+
+    Parameters
+    ----------
+    date :
+        Date for the output header.
+    observer_coordinate :
+        Observer coordinate for the output header.
+    shape : [int, int]
+        Output map shape, number of pixels in (latitude, longitude).
+    projection_code : {'CAR', 'CEA'}
+        Projection to use for the latitude coordinate.
+
+    Returns
+    -------
+    `~sunpy.util.MetaDict`
+    """
+    valid_codes = {"CAR", "CEA"}
+    if projection_code not in valid_codes:
+        raise ValueError(f"projection_code must be one of {valid_codes}")
+
+    frame_out = SkyCoord(
+        0 * u.deg,
+        0 * u.deg,
+        frame="heliographic_carrington",
+        obstime=date,
+        observer=observer_coordinate,
+    )
+
+    if projection_code == "CAR":
+        scale = [360 / int(shape[1]), 180 / int(shape[0])] * u.deg / u.pix
+    elif projection_code == "CEA":
+        # Using the cylindrical equal-area (CEA) projection,
+        # scale needs to be to 180/pi times the sin(latitude) spacing
+        # See Section 5.5, Thompson 2006
+        scale = [
+            360 / int(shape[1]),
+            (180 / np.pi) / (int(shape[0]) / 2)
+        ] * u.deg / u.pix
+
+    # Header helper expects shape to be in [y, x] order, but scale in [x, y]...
```

Comment:
This is now out of date?
```suggestion
```

Reply:
Good catch - will remove alonside any bikeshedding that gets done above 👍